autoPyTorch/pipeline/tabular_classification.py

```diff
@@ -289,6 +289,29 @@ def _get_hyperparameter_search_space(self,
                                 raise ValueError("Cannot find a legal default configuration")
                             cs.get_hyperparameter('network_embedding:__choice__').default_value = default
 
+        # Disable CyclicLR until todo is completed.
+        if 'lr_scheduler' in self.named_steps.keys() and 'trainer' in self.named_steps.keys():
+            trainers = cs.get_hyperparameter('trainer:__choice__').choices
+            for trainer in trainers:
+                available_schedulers = self.named_steps['lr_scheduler'].get_available_components(
+                    dataset_properties=dataset_properties,
+                    exclude=exclude if bool(exclude) else None,
+                    include=include if bool(include) else None)
```

Comment:
yes because our `get_available_components` logic checks if `include` or `exclude` are `None` or not. I think though we dont need to use `get_available_components` as I only need the string names. I'll fix this.